src/bundle/Controller/LocationController.php

```diff
@@ -485,6 +483,9 @@ public function updateVisibilityAction(Request $request): Response
         if ($form->isSubmitted() && $form->isValid()) {
             $result = $this->submitHandler->handle($form, function (LocationUpdateVisibilityData $data) use ($contentInfo) {
                 $location = $data->getLocation();
+
+                $this->canUserHideLocation($location);
```

Comment:
Don't use "breakpoint"-like methods (no return value with exception in case of failure). There is need to wrap this simple `if` here.